File: Day-02 ( Swap Two Numbers Using a Temporary Variable ) /CPP/Solution.cpp

// For Both a And b In Same Line.    

#include <iostream>
using namespace std;

int main() {
    int a, b, temp;

    cin >> a >> b;

    cout << "Before swap: a = " << a << ", b = " << b << endl;

    // Swap using temporary variable
    temp = a;
    a = b;
    b = temp;

    cout << "After swap: a = " << a << ", b = " << b << endl;

    return 0;
}









// For Both a And b In Different Line.  

#include <iostream>
using namespace std;

int main() {
    int a, b, temp;

    cin >> a;
    cin >> b;

    cout << "Before swap: a = " << a << ", b = " << b << endl;

    // Swap using temporary variable
    temp = a;
    a = b;
    b = temp;

    cout << "After swap: a = " << a << ", b = " << b << endl;

    return 0;
}

File: Day-02 ( Swap Two Numbers Using a Temporary Variable ) /Python/Solution.py

# For Both a And b in Same Line. 
# a, b = map(int, input().split()) 
# expects both numbers in one line. 


a, b = map(int, input().split())

print(f"Before swap: a = {a}, b = {b}")

temp = a
a = b
b = temp

print(f"After swap: a = {a}, b = {b}")












# For Both a And b In Different Line. 

a = int(input())
b = int(input())

print(f"Before swap: a = {a}, b = {b}")

temp = a
a = b
b = temp

print(f"After swap: a = {a}, b = {b}")




